xlive/Blam/Engine/main/main_game.cpp

```diff
@@ -0,0 +1,213 @@
+#include "stdafx.h"
+#include "main_game.h"
+
+#include "Blam/Engine/cache/cache_files.h"
+#include "Blam/Engine/cseries/cseries.h"
+#include "Blam/Engine/cseries/cseries_strings.h"
+#include "Blam/Engine/game/game.h"
+#include "Blam/Engine/Networking/Session/NetworkSession.h"
+#include "Blam/Engine/saved_games/game_variant.h"
+
+#include "Util/Hooks/Hook.h"
+
+s_game_options g_main_game_launch_options;
+DWORD g_main_game_launch_user_count;
```

Comment:
Nit int

Reply:
the two globals are initialized now.

Reply:
Meant you should make the variable int instead.